roles/download/defaults/main.yml

```diff
@@ -434,6 +445,24 @@ nerdctl_archive_checksums:
   amd64:
     0.8.1: 7c3573db282749079e06f4c592b4585d53628d13fd762746b30389f854d79a47
 
+# TODO(cristicalin): remove compatibility entries once debian9 and ubuntu16 jobs are dropped or docker is dropped
+containerd_archive_checksums:
+  arm:
+    1.4.3: 0  # to make debian9 jobs happy
+    1.4.6: 0  # to make ubuntu16 jobs happy
+    1.4.9: 0
+    1.5.5: 0
+  arm64:
+    1.4.3: 0  # to make debian9 jobs happy
+    1.4.6: 0  # to make ubuntu16 jobs happy
+    1.4.9: 0
+    1.5.5: 0
+  amd64:
+    1.4.3: 0  # to make debian9 jobs happy
+    1.4.6: 0  # to make ubuntu16 jobs happy
```

Comment:
If deploying with `container_engine=containerd` this would not be needed, but when deploying with `container_engine=docker` the older versions of docker supported on those platforms bring in dependencies for older containerd and would break the CI runs due to the way the `download` role does variable interpolation.

We could also add `latest` instead of the specific versions and set `containerd_version=latest` in the debian9 and ubuntu16 jobs.

Reply:
I just pushed a change with using `latest` instead of specific versions, at least this keeps the code a bit cleaner.
Let's see if the CI gods agree.